docs/docs/other/plans.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: "Billing"
+title: "Plans Explained"
```

Comment:
Nit: we should use "Billing Plan" not just "Plan" (in the slug, sidebar, and title)